docs/zh_cn/common_usage/model_analysis.md

```diff
@@ -0,0 +1,740 @@
+# 统计模型计算量和参数量
+
+1. 定义模型
+
+   ```python
+   import torch.nn.functional as F
+   import torchvision
+   from mmengine.model import BaseModel
+   from mmengine.analysis import get_model_complexity_info
+
+
+   class MMResNet50(BaseModel):
+       def __init__(self):
+           super().__init__()
+           self.resnet = torchvision.models.resnet50()
+
+       def forward(self, imgs, labels=None, mode='tensor'):
+           x = self.resnet(imgs)
+           if mode == 'loss':
+               return {'loss': F.cross_entropy(x, labels)}
+           elif mode == 'predict':
+               return x, labels
+           elif mode == 'tensor':
+               return x
+   ```
+
+2. 统计计算量和参数量
+
+   ```python
+   input_shape = (3, 224, 224)
+   model = MMResNet50()
+   analysis_results = get_model_complexity_info(model, input_shape)
+   ```
+
+- 以表格的形式显示
+
+  ```python
+  print(analysis_results['out_table'])
+  ```
+
+  <details>
+    <summary>点击展开</summary>
+
+  ```
+    +------------------------+----------------------+------------+--------------+
```

Comment:
These are some strange indents on the table. Is this expected?

Reply:
done